sdk/eventhub/event-hubs/samples-dev/sendBufferedEvents.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/**
+ @summary Demonstrates how to send events to an Event Hub using the `EventHubBufferedProducerClient`.
+ * This sample is different from the one in `sendEvent.ts` in that the client manages batching of events and sending
+ * after a given amount of time or after a given amount of events are in a batch instead of you managing the same explicitly.
+ *
+ * @azsdk-weight 60
+ */
+
+import { AmqpAnnotatedMessage } from "@azure/core-amqp";
+import {
+  EventData,
+  EventHubBufferedProducerClient,
+  OnSendEventsErrorContext,
+} from "@azure/event-hubs";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env["EVENTHUB_CONNECTION_STRING"] || "";
+
+async function handleError(ctx: OnSendEventsErrorContext): Promise<void> {
+  console.log(`The following error occurred:`);
+  console.log(JSON.stringify(ctx.error, undefined, "  "));
+  console.log(
+    `The following events were not sent as a result to the partition with ID ${ctx.partitionId}:`
+  );
+  console.log(
+    ctx.events
```

Comment:
I'd rewrite this as a `for (const event of ctx.events)` rather than `.map(...).join`.

If we're talking about implementation code I'd probably write it the FP way as well, but for the samples I think the for-of is more natural.

Reply:
Done!